### PR TITLE
[17.0][MIG] stock_demand_estimate

### PR DESCRIPTION
**Pull request by CarlosVForgeFlow**
Migration to 17.0

@ForgeFlow

### CODE REVIEW

File: stock_demand_estimate/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {
     "name": "Stock Demand Estimate",
     "summary": "Allows to create demand estimates.",
-    "version": "16.0.1.1.0",
+    "version": "17.0.1.1.0",
```
Comment (rousseldenis) [inline stock_demand_estimate/__manifest__.py]: @CarlosVForgeFlow Could you reset to 17.0.1.0.0 ?

Comment (CarlosVForgeFlow) [reply stock_demand_estimate/__manifest__.py]: Done!

**Review by AndreuOForgeFlow — APPROVED** [review]
Code review

**Review by LoisRForgeFlow — APPROVED** [review]
Functional test in runbot :+1: 

/ocabot merge nobump

**Pull request merged**